src/Tab.vue

```diff
@@ -21,6 +21,10 @@ export default {
     disabled: {
       type: Boolean,
       default: false
+    },
+    customClass: {
```

Comment:
```html
 <tab header="First tab" custom-class="meow woof doot">
    ...
  </tab>
  <tab header="Disabled second tab :x:" disabled custom-class="meow woof doot"> 
  </tab>
```

Tabs and TabGroups doesn't seem to have the correct classes, they did not appear in the html when I checked it.